src/main/java/at/hannibal2/skyhanni/features/dungeon/DungeonSecretChime.kt

```diff
@@ -0,0 +1,36 @@
+package at.hannibal2.skyhanni.features.dungeon
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.api.event.HandleEvent
+import at.hannibal2.skyhanni.data.ClickedBlockType
+import at.hannibal2.skyhanni.events.DungeonBlockClickEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.SoundUtils
+import at.hannibal2.skyhanni.utils.SoundUtils.playSound
+
+@SkyHanniModule
+object DungeonSecretChime {
+    private val config get() = SkyHanniMod.feature.dungeon.secretChime
+
+    @HandleEvent
+    fun onDungeonClickedBlock(event: DungeonBlockClickEvent) {
+        if (!isEnabled()) return
+        val isWaterRoom = DungeonAPI.getRoomID() == "-60,-60"
+        if (isWaterRoom && event.blockType == ClickedBlockType.LEVER) return
+
+        val type = event.blockType
+        when (type) {
+            ClickedBlockType.CHEST, ClickedBlockType.TRAPPED_CHEST, ClickedBlockType.LEVER, ClickedBlockType.WITHER_ESSENCE -> playSound()
+        }
```

Comment:
add an else since when more get added later on to the ClickedBlockType someone needs to fix this. Since when's of this type need to be exhaustive. Which it is now, since every value is listed but in the future it will error. 
```suggestion
        when (type) {
            ClickedBlockType.CHEST, ClickedBlockType.TRAPPED_CHEST, ClickedBlockType.LEVER, ClickedBlockType.WITHER_ESSENCE -> playSound()
            else -> {}
        }
```